apollo-router/src/query_planner/bridge_query_planner_pool.rs

```diff
@@ -119,21 +127,64 @@ impl BridgeQueryPlannerPool {
             });
         }
         let sender_for_gauge = sender.clone();
-        let pool_size_gauge = meter_provider()
-            .meter("apollo/router")
+        let meter = meter_provider().meter("apollo/router");
+        let pool_size_gauge = meter
             .u64_observable_gauge("apollo.router.query_planning.queued")
```

Comment:
I know it existed before but could you add description and unit please ?

Reply:
what kind of unit do we set here? Is there a convention for the expected units in otel?

Reply:
(asking because I don't see many examples of setting units in our metrics)

Reply:
Yes there are. At different places but for example for memory you can find an example here https://opentelemetry.io/docs/specs/semconv/system/hardware-metrics/#hwmemory---memory-module-metrics and more generally you can take examples from [here](https://opentelemetry.io/docs/specs/semconv/general/metrics/) there are some conventions too

Reply:
For this on I would suggest using `query`

Reply:
done